app/main/constants/config-const.js

```diff
@@ -6,7 +6,7 @@ angular.module('main')
   ENV: {
     /*inject-env*/
     'SERVER_URL': 'http://sample-env.qtbvngb2iz.us-west-2.elasticbeanstalk.com/',
```

Comment:
Actually, in this file, we don't have to write anything. In this same directory, there are 2 more JSON files. One for prod and one for dev. Depending on the build env it inject the corresponding file's data.